index.ts

```diff
@@ -1,3 +1,4 @@
 export * from "./typechain";
 export * from "./src/DeploymentUtils";
 export * from "./utils";
+export * from "./src/svm/assets";
```

Comment:
The program getters, types, and IDLs can be imported directly from `"@across-protocol/contracts"`, but we don’t import the SVM utility functions because a subset of these should be shared in the SDK.